core/ir/aarch64/instr_create_api.h

```diff
@@ -463,12 +463,12 @@
  * they just need to know whether they need to preserve the app's flags, so maybe
```

Comment:
Please update the PR description to describe how you tested this, since there are no tests added/enabled by this PR (see prior comment which will add some).